Lesson24/deniz/newsletter assignment/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Newsletter sign-up form</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="card">
+        <div class="right">
+            <h1>Stay updated!</h1>
+            <p>Join 60,000+ product managers receiving <br>monthly updates on:</p>
+            <ul>
+                <li>Product discovery and building what matters</li>
+                <li>Measuring to ensure updates are a success</li>
+                <li>And much more!</li>
+            </ul>
+            <form>
+                <label for="email"><strong>Email Address</strong></label>
+                <input type="email" id="email" placeholder="email@company.com">
+                <button type="submit">Subcribe to Monthly Newsletter</button>
+            </form>
+        </div>
+        <div class="left">
+            <img src="./assets/img_desktop.jpg" alt="a designed image">
+        </div>
+    </div>
+
+    <div class="card-2 hide">
+        <img src="./assets/icon-success.svg" class="icon-card2" alt="">
+        <h1>Thanks for subscribing!</h1>
+        <p>
+            A confirmation email has been sent to<strong> <a
```

Comment:
Avoid using <strong> tags, use CSS for styling instead

Reply:
do you mean seizing the svg?